whitepaper.tex

```diff
@@ -170,18 +170,20 @@ \subsection{Route metric verification}
 
 \subsection{Verification scheduling}
 \begin{unbreakable}
-We have the ability to test and verify the routes advertised by neighbors, but we need some way of deciding which routes to verify. Verification runs on a timer. Each verification cycle a node follows this procedure to choose a route $r$ to verify:
+We have the ability to test and verify the routes advertised by neighbors, and once we have connections open to endpoints we are already using piggybacking packets to verify already used connections is trivial and can be done continuously to prevent fraud. But we would also like to avoid participating in the advertisement of false routes unkowningly. Therefore we need some way of deciding routes to verify from the set of routes in the local routing table we are not currently using.
```

Comment:
I think this more closely reflects what we actually plan to do. So I guess that's a yes.